reference/configuration/framework.rst

```diff
@@ -596,6 +597,23 @@ If you're using forms, but want to avoid starting your session (e.g. using
 forms in an API-only website), ``csrf_protection`` will need to be set to
 ``false``.
 
+.. _config-framework-error_controller:
+
+error_controller
+~~~~~~~~~~~~~~~~~~~~
+
+**type**: ``string`` **default**: ``error_controller``
+
+This is the controller that is activated after an exception is thrown anywhere
+in your application. The default controller
+(:class:`Symfony\\Component\\HttpKernel\\Controller\\ErrorController`)
+is what's responsible for rendering specific templates under different error
+conditions (see :doc:`/controller/error_pages`). Modifying this
+option is advanced. If you need to customize an error page you should use
+the previous link. If you need to perform some behavior on an exception,
+you should add an :doc:`event listener </event_dispatcher>` to the
+:ref:`kernel.exception event <kernel-kernel.exception>`.
```

Comment:
I would say we can remove anything after "Modifying this option is advanced". The error_pages document seems to explain anything that's explained in the sentences after it in a much clearer way, doesn't it? (I know you copy-pasted this from below, I was just thinking whether we can improve this)